### PR TITLE
Adding logic for client correlation id in get api

### PR DESCRIPTION
**Pull request by ankita10r**
## Description

*The PR includes following changes:

- X-CorrelationId to be added to Ops App Get TxnRequest API
- X-CorrelationId to be added to Ops App Export CSV TxnRequest API 
- Contains Ops App part of X-CorrelationId to be added to channel new API (TxnState by ClientCorrelationId)[The new api internally calls /transfer api 
Channel new API: https://github.com/openMF/ph-ee-connector-channel/pull/50

## Checklist

Please make sure these boxes are checked before submitting your pull request - thanks!

- [X] Design related bullet points or design document link related to this PR added in the description above. 

- [X] Updated corresponding Postman Collection or Api documentation for the changes in this PR. _(Pending PR)_

- [ ] Create/update unit or integration tests for verifying the changes made._(Pending PR)_

- [ ] Add required Swagger annotation and update API documentation with details of any API changes if applicable

- [X] Followed the naming conventions as given in https://docs.google.com/document/d/1Q4vaMSzrTxxh9TS0RILuNkSkYCxotuYk1Xe0CMIkkCU/edit?usp=sharing

### CODE REVIEW

File: src/main/java/org/apache/fineract/api/OperationsDetailedApi.java [hunk 1]
```diff
@@ -192,6 +197,10 @@ public Page<TransactionRequest> transactionRequests(
         if (currency != null) {
             specs.add(TransactionRequestSpecs.match(TransactionRequest_.currency, currency));
         }
+        if (clientCorrelationId != null) {
+            logger.info("clientCorrelationId: " + clientCorrelationId);
```
Comment (fynmanoj) [inline src/main/java/org/apache/fineract/api/OperationsDetailedApi.java:201]: use debug log, 
donot use +

File: src/main/java/org/apache/fineract/operations/TransactionRequest.java [hunk 2]
```diff
@@ -76,6 +76,10 @@ public class TransactionRequest extends AbstractPersistableCustom<Long> {
 
     @Column(name = "EXTERNAL_ID")
     private String externalId;
+
+    @Column(name = "CLIENTCORRELATIONID")
```
Comment (fynmanoj) [inline src/main/java/org/apache/fineract/operations/TransactionRequest.java:80]: use the same format in names as CLIENTCORRELATION_ID

**Review by fynmanoj — CHANGES_REQUESTED** [review]

File: src/main/java/org/apache/fineract/operations/TransactionRequest.java (hunk 2 above)
Comment (ankita10r) [reply src/main/java/org/apache/fineract/operations/TransactionRequest.java:80]: Based on this PR https://github.com/openMF/ph-ee-operations-app/pull/43/files that is merged, if i change the value here i would need to update it all places, should i do it?

Comment (fynmanoj) [reply src/main/java/org/apache/fineract/operations/TransactionRequest.java:80]: lets keep it this way as there is a dependency now.

**Pull request merged**